src/api.js

```diff
@@ -24,7 +24,9 @@ export const getConferenceById = async id => {
 export const createNewConference = async conference => {
   try {
     const response = await axios.post(
-      "http://localhost:8080/conferences",conference);
+      "http://localhost:8080/conferences",
+      conference
```

Comment:
should conference not be wrapped in {} as body is an object?

Reply:
We have tried with the bracket added in, but it stops the API request working?

Reply:
Conference is an object so it should be fine actually, ignore me 😅